system-test/Makefile

```diff
@@ -25,22 +25,43 @@ J ?= 1
 # directory at GLOBAL_LUSTRE_ROOT (e.g. /lus/global/myuser).
 GLOBAL_LUSTRE_ROOT ?= /lus/global
 
+# Default to no flux queue
+Q ?=
+
+# Default bats command
+BATS = bats -j $(J)
+
 # add bats to PATH
 export PATH := $(shell ./bats_env.sh):$(PATH)
 
-.PHONY: all init dm clean env
-
+.PHONY: all
 all: init test dm
 
+.PHONY: init
 init:
 	./bats_install.sh
 
+.PHONY: sanity
+sanity:
+	$(BATS) --filter-tags tag:sanity .
+
+.PHONY: simple
+simple:
+	$(BATS) --filter-tags tag:simple .
+
+.PHONY: %
+%:
```

Comment:
I had to do some digging. I haven't seen a bare '%' like this.
This looks like a last-resort rule: https://www.gnu.org/software/make/manual/make.html#Last-Resort

Change the colon to double-colon as shown in that section of the manual.

The .PHONY is not doing what you intend, so it should be removed. See https://www.gnu.org/software/make/manual/make.html#Phony-Targets , a couple of paragraphs in: "Prerequisites of .PHONY are always interpreted as literal target names, never as patterns (even if they contain ‘%’ characters). "